clouddrift/binning.py

```diff
@@ -33,12 +329,21 @@ def binned_statistics(
         - None: defaults to 10 bins per dimension.
     bins_range : list of tuples, optional
         Outer bin limits for each dimension.
+    statistics : str or list of str, Callable[[np.ndarray], float] or list[Callable[[np.ndarray], float]]
+        Statistics to compute for each bin. It can be:
+        - a string, supported values: 'count', 'sum', 'mean', 'median', 'std', 'min', 'max', (default: "count"),
+        - a custom function as a callable that take a 1D array of values and return a single value.
+          The callable is applied at each variable.
+        - a tuple of (var_name, callable) for multivariate statistics. 'var_name' is used to identify the resulting variable.
+          In this case, the callable will receive the list of arrays provided in `data`. For example, to calculate kinetic energy,
+          you can pass `data = [u, v]` and  `statistics=("ke", lambda data: np.sqrt(np.mean(data[0] ** 2 + data[1] ** 2)))`.
+        - a list containing any combination of the above, e.g., ['mean', np.nanmax, ('ke', np.sqrt(np.mean(data[0] ** 2 + data[1] ** 2)))].
```

Comment:
Do you mean:  - a list containing any combination of the above, e.g., ['mean', np.nanmax, ('ke', lambda data: np.sqrt(np.mean(data[0] ** 2 + data[1] ** 2)))].